### DISCUSSION TITLE
dimming below 0%

### DISCUSSION THREAD

**Discussion by llll-jase** [xanderfrangos/twinkle-tray#347] [Ideas]
Thanks for this great little app!

Unfortunately, many PC monitors on their lowest backlight are still too bright. On macos, there’s an app called MonitorControl that combines ddc/ci with a black overlay once the backlight reaches 0%, giving the illusion that the screen can go even dimmer. Their implementation/design is superb. Although there are windows apps that do black overlays (e.g., DimScreen, f.lux), they’re not as seamless or visually appealing and don’t do backlight control.

I believe this would be a useful and appreciated addition to Twinkle Tray, making it a modern all-in-one solution for brightness control on PC.

A few things that would be nice:
- Ability to dim cursor with the screen (fixes “white cursor”)
- Ability to assign hotkeys (or a similar convenience feature)
- Ability to have separate sliders for backlight and overlay OR merge into one continuous slider, which would be lovely ☺️ (e.g., 0% to 50% is overlay control, 50% to 100% is backlight control, indication/mark to clearly distinguish)

**Comment by nicholasvafiades** [comment]
Having a continuous slider partitioned between overlay and backlight from 0-100 may be confusing since hardware brightness is always 0-100, but maybe there could be the ability to go negative which represents when hardware brightness is at 0 and an overlay is enabled. So the range could be -20% to 100%, with -20% to 0% representing 0 hardware brightness and 0-20 range of overlay.
